src/components/RecNode/RecNode.tsx

```diff
@@ -77,4 +66,71 @@ const RecNodeField: FC<RecNodeFieldProps> = ({ title, description }) => {
   );
 };
 
+function renderRecGuidance(pathwayState: GuidanceState): ReactElement[] {
+  const resource = pathwayState.action[0].resource;
+  const coding =
+    'medicationCodeableConcept' in resource
+      ? resource.medicationCodeableConcept.coding
+      : resource.code.coding;
+  return [
+    <RecNodeField key="Notes" title="Notes" description={pathwayState.action[0].description} />,
+    <RecNodeField key="Type" title="Type" description={resource.resourceType} />,
+    <RecNodeField
+      key="System"
+      title="System"
+      description={
+        <>
+          {coding[0].system}
+          <a href={coding[0].system} target="_blank" rel="noopener noreferrer">
+            <FontAwesomeIcon icon="external-link-alt" className={classes.externalLink} />
+          </a>
+        </>
+      }
+    />,
+    <RecNodeField key="Code" title="Code" description={coding[0].code} />,
+    <RecNodeField key="Display" title="Display" description={coding[0].display} />
+  ];
+}
+function parseResource(resource: fhir.DomainResource): ReactElement[] {
```

Comment:
This looks easy enough to append to when needed but is there anything else we know should be included right now? The observation should probably include the system/code/display at the least

Reply:
I can think of two things but not sure how we'll be modeling one of them. First one is MedicationRequests - we'd want the prescription date and status of medications. Second thing is the note that gets added, but we don't know the structure of that just yet. Beyond that, like you said anything else can be added pretty easily.

Reply:
I'm going to leave additional fhir info for another task

Reply:
Can you handle MedicationRequests here since that's going to be pretty common and you're already working this file? If we get that we probably won't need much more.